packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1794,20 +1797,24 @@ class Flex extends MultiChildRenderObjectWidget {
   /// How the children should be placed along the main axis.
   final MainAxisAlignment mainAxisAlignment;
 
+  /// The limit used to compute free space along the main axis.
+  final MainAxisSpace mainAxisSpace;
+
   /// How the children should be placed along the cross axis.
   final CrossAxisAlignment crossAxisAlignment;
 
   /// If aligning items according to their baseline, which baseline to use.
   final TextBaseline textBaseline;
 
   @override
-  RenderFlex createRenderObject(BuildContext context) => new RenderFlex(direction: direction, mainAxisAlignment: mainAxisAlignment, crossAxisAlignment: crossAxisAlignment, textBaseline: textBaseline);
+  RenderFlex createRenderObject(BuildContext context) => new RenderFlex(direction: direction, mainAxisAlignment: mainAxisAlignment, mainAxisSpace: mainAxisSpace, crossAxisAlignment: crossAxisAlignment, textBaseline: textBaseline);
```

Comment:
I'd break this over multiple lines at this point and switch from `=>` to `{` / `}`